src/main/java/org/venice/beachfront/bfapi/database/dao/DetectionDao.java

```diff
@@ -0,0 +1,17 @@
+package org.venice.beachfront.bfapi.database.dao;
+
+import java.util.List;
+import javax.transaction.Transactional;
+
+import org.springframework.data.repository.CrudRepository;
+import org.venice.beachfront.bfapi.model.Detection;
+import org.venice.beachfront.bfapi.model.DetectionPK;
+
+@Transactional
+public interface DetectionDao extends CrudRepository<Detection, DetectionPK> {
+	Detection findByJob_JobIdAndFeatureId(String jobId, int featureId);
```

Comment:
Is this a valid method name?  Have you tested it?

Reply:
I have not tested actually persisting and querying data, but I can do that before we merge. In theory, (and according to their documentation) it should work.